### ISSUE TITLE
bug: $limit in resources /resource/{id}

### ISSUE THREAD

**Issue by marcosanz-apiquality** [apiaddicts/sonaropenapi-rules#43]
# Bug Report

In the rule OAR025, check the parameter $limit in resources /resource/{id}. It's wrong, because one resource never would has pagination.

**Current behavior:**

Don't show errors in those resources

**Expected behavior:**

**Steps to reproduce:**

Upload a openapi without $limit parameter in a resource /resouce/{id} and activate rule OAR025.

**Related code:**

```
insert short code snippets here
```

**Other information:**
